src/theme/components.js

```diff
@@ -191,6 +191,24 @@ const components = {
       },
     },
   },
+  MuiContainer: {
+    styleOverrides: {
+      root: {
+        '&.perfcompare-body': {
+          minHeight: '85vh',
+        },
+      },
+    },
+  },
+  MuiAlert: {
+    styleOverrides: {
+      root: {
+        '&.feedback-alert': {
+            backgroundImage: 'linear-gradient( 45deg, rgb(138,35,135) 10%, rgb(233,64,87) 50%, rgb(242,113,33) 100% )',
```

Comment:
hey Andra, I didnt compile the code but does this infer that you're drawing an object here?
Would using an asset be good ?

Reply:
This is used to create a gradient background for the Alert, similar colors to the underline for the PerfCompare header, I left it so I get an opinion on how it looks- if not appropriate I'll change it back, this is why I added a separate commit for it.